core/src/components/select/test/compare-with/select.e2e.ts

```diff
@@ -21,4 +21,52 @@ test.describe('select: compare-with', () => {
       value: '1',
     });
   });
+
+  test('should pass compareWith params in the correct order', async ({ page }, testInfo) => {
```

Comment:
~ This description makes the test seem hyper-specific to the point of not being all that useful; could we generalize it a bit more? Maybe something like "should work with mismatched types"?

Reply:
fixed in  https://github.com/ionic-team/ionic-framework/pull/25764/commits/caa25275fafbdd507a081e87616a577423fa1a5d